advanced-usage: Add bag editor (#75)
bluerobotics/ardusub-zola: advanced-usage/index.md

{% end %}
{{ easy_image(src="available-services", width=600, class="pirate") }}

{% pirate() %}

> ### Bag Editor

Reviewer: ```suggestion
### Bag Editor
{{ service(service="Bag of Holding", port=9101, link="/services/bag_of_holding", based=true) }}
```

> The Bag Editor is a helper service for advanced users, with it is possible to change the

Reviewer: ```suggestion
The Bag Editor is a helper service for advanced users, which allows modifying the
```

database used to handle frontend interface changes.
{% end %}

> {{ easy_image(src="bag-editor", width=600, class="pirate") }}

Reviewer: You've linked to an image here, but seemingly forgot to include the actual image in the PR?
